core/helpers/EEH_Activation.helper.php

```diff
@@ -1510,96 +1562,99 @@ public static function drop_espresso_tables()
-            'ee_id_mapping_from'                 => false,
-            'espresso_persistent_admin_notices'  => true,
-            'ee_encryption_key'                  => true,
-            'pue_force_upgrade_'                 => false,
-            'pue_json_error_'                    => false,
-            'pue_install_key_'                   => false,
-            'pue_verification_error_'            => false,
-            'pu_dismissed_upgrade_'              => false,
-            'external_updates-'                  => false,
-            'ee_extra_data'                      => true,
-            'ee_ssn_'                            => false,
-            'ee_rss_'                            => false,
-            'ee_rte_n_tx_'                       => false,
-            'ee_pers_admin_notices'              => true,
-            'ee_job_parameters_'                 => false,
-            'ee_upload_directories_incomplete'   => true,
-            'ee_verified_db_collations'          => true,
-        );
+            'ee_config'                                  => false,
+            'ee_data_migration_current_db_state'         => true,
+            'ee_data_migration_mapping_'                 => false,
+            'ee_data_migration_script_'                  => false,
+            'ee_data_migrations'                         => true,
+            'ee_dms_map'                                 => false,
+            'ee_notices'                                 => true,
+            'lang_file_check_'                           => false,
+            'ee_maintenance_mode'                        => true,
+            'ee_ueip_optin'                              => true,
+            'ee_ueip_has_notified'                       => true,
+            'ee_plugin_activation_errors'                => true,
+            'ee_id_mapping_from'                         => false,
+            'espresso_persistent_admin_notices'          => true,
+            'ee_encryption_key'                          => true,
+            'pue_force_upgrade_'                         => false,
+            'pue_json_error_'                            => false,
+            'pue_install_key_'                           => false,
+            'pue_verification_error_'                    => false,
+            'pu_dismissed_upgrade_'                      => false,
+            'external_updates-'                          => false,
+            'ee_extra_data'                              => true,
+            'ee_ssn_'                                    => false,
+            'ee_rss_'                                    => false,
+            'ee_rte_n_tx_'                               => false,
+            'ee_pers_admin_notices'                      => true,
+            'ee_job_parameters_'                         => false,
+            'ee_upload_directories_incomplete'           => true,
+            'ee_verified_db_collations'                  => true,
```

Comment:
So we still do these indents ? Asking because I remember when we first changed our coding standards we made a decision to drop this. So I usually don't do these indents.

Reply:
oh my IDE is doing it automatically...
I can switch back... just have to find that setting...
unless you prefer aligned elements like this and want to switch over...
I can't remember who was opposed to this and/or what the reason might be to not do it!

Reply:
PhpStorm ?
I'm not opposed of this at all. It's just that I'm not sure now if I should just do the same. Because a year ago I remember I was removing all those spaces.

Reply:
Yes I'm using PhpStorm. What are you using?

My settings got reset recently because I just built a new system and reinstalled all of my software, so I'll have to find the setting for turning off that alignment. I do prefer having all of the array elements aligned because I think it looks neater and is easier to read, but I don't feel that strongly about it. If your IDE can do that automatically and you also prefer aligned elements, then let's change, but if you have to add those spaces manually, then don't bother. I will find the option in my settings and turn that off, because having to manually align things is not worth the benefit.

So I'll let you decide. Don't worry about what the option was in the past, just decide what you would prefer going forward.

Reply:
> Yes I'm using PhpStorm. What are you using?

Same.

> I do prefer having all of the array elements aligned because I think it looks neater and is easier to read

Agree. It's just that one time when there's waay too many spaces on some short lines, then it looks a bit overwhelming. But yes, we can stick with this, I'm partly for it.
I'm using the EE settings from our GH repo, with a few of my own changes. But it never added those indents automatically, so I'll have to find that option and enable it myself.